### DISCUSSION TITLE
[Bug] Input field not resizing when wrap in ui labeled input

### DISCUSSION THREAD

**Discussion by YauKeng** [fomantic/Fomantic-UI#1448] [Elements (button, image, label, loader, ...)] [answered]
# Bug Report
`input` not resizing when wrap in `ui labeled input`

## Steps to reproduce

1. Open [jsFiddle](https://jsfiddle.net/46u2p91x/3/)
2. run and drag the result screen smaller (below 330px).
3. See the Name and Price field.

## Expected result
Input field resize to smaller like others (textarea, dropdown)

## Actual result
Input not resizing.

## Testcase

https://jsfiddle.net/46u2p91x/3/
## Screenshot (if possible)

[Image](https://res.cloudinary.com/myfoodkampung/image/upload/v1588492577/MyFoodKampung/pua17bfpdizpcuhvj2wa.jpg)
## Version

2.8.4

**Comment by lubber-de** [comment]
Mmh, the browser does not seem to calculate the input width correctly because of the `auto` being used.

It's very hacky but you can quickly fix this by forcing a much smaller width for the input element itself.

```css
.ui.ui.ui.labeled.input > input {
  width:10px;
}
```
See https://jsfiddle.net/7kd5wsc4/

**Comment by jamessampford** [comment, marked as answer]
Isn’t it a case of changing it from `ui labeled input` to `ui labeled fluid input`?

**Comment by lubber-de** [comment]
> Isn’t it a case of changing it from `ui labeled input` to `ui labeled fluid input`?

Indeed 👍 
See https://jsfiddle.net/gfzcxy2d/

and forget my previous comment 🙄 😄
